configsrc/vcluster/main/default_values.yaml

```diff
@@ -1,863 +1,499 @@
-# Sync describes how to sync resources from the virtual cluster to host cluster and back.
+# DO NOT ADD ANY COMMENTS TO THIS FILE.
+# Comments are added automatically in the hack/schema/main.go script according to the type defined in config.go
+# If you want to change or add any comment, please change/add it in the config.go and rerun hack/schema/main.go
```

Comment:
@Piotr1215 / @joaniekube  all the comments getting removed from this file are on purpose then ? 
The normal file still has all of them: https://github.com/loft-sh/vcluster/blob/main/chart/values.yaml#L3

Reply:
I think this is only for the config file for the auto-generated process and not for values.yaml. The comments in values.yaml should still be as they are.